projects/packages/jetpack-mu-wpcom/package.json

```diff
@@ -62,6 +66,7 @@
 		"@wordpress/url": "4.2.0",
 		"@wordpress/icons": "10.2.0",
 		"clsx": "2.1.1",
+		"i18n-calypso": "7.0.0",
```

Comment:
Would there be a way around bringing the whole of `i18n-calypso` to the mu-wpcom package? It was nice that we were able to stick to `@wordpress/i18n` until now, it would be great if we could continue to do so.

I'm not familiar enough with `i18n-calypso` to know if we could work around that dependency though.

Reply:
It would be better to get rid of the `i18n-calypso` dependency from the `@automattic/i18n-utils` package and use `@wordpress/i18n` instead.

It seems to be done by just removing the usage of the `getLocaleSlug` call from the `@automattic/i18n-utils` package.

Reply:
I've remove the `i18n-calypso` dependency from the `@automattic/i18n-utils`. See https://github.com/Automattic/wp-calypso/pull/92631.